Dockerfile

```diff
@@ -1,10 +1,39 @@
-FROM golang
+# Build the spiffe-helper binary
+FROM --platform=$BUILDPLATFORM golang:1.21.5-alpine as base
```

Comment:
may we get go version from makefile? or variable? to reduce the amount of places we need to update in case we are upgrading golang version

Reply:
Done